pypet2bids/README.md

```diff
@@ -0,0 +1,132 @@
+build process.
+
+After installing poetry, you can build and install this package to your local version of Python with the following 
+commands (keep in mind the commands below are executed in a bash-like shell):
+
+```bash
+cd PET2BIDS
+cp -R metadata/ pypet2bids/pypet2bids/metadata
+cp pypet2bids/pyproject.toml pypet2bids/pypet2bids/pyproject.toml
+cd pypet2bids && poetry lock && poetry build
+pip install dist/pypet2bids-X.X.X-py3-none-any.whl
+```
+
+Why is all the above required? Well, because this is a monorepo and we just have to work around that sometimes.
+
+
+[!NOTE]
+Make and the additional scripts contained in the `scripts/` directory are for the convenience of 
+non-windows users.
+
+If you have GNU make installed and are using a bash or something bash-like in you your terminal of choice, run the 
+following:
+
+```bash
+cd PET2BIDS
+make installpoetry buildpackage installpackage
+```
+
+</details>
+
+<details> 
+<summary>Run Directly From Source</summary>
+
+Lastly, if one wishes run pypet2bids directly from the source code in this repository or to help contribute to the python portion of this project or any of the documentation they can do so via the following options:
+
+```bash
+cd PET2BIDS/pypet2bids
+poetry install
+```
+
+Or they can install the dependencies only using pip:
+
+```bash
+cd PET2BIDS/pypet2bids
+pip install .
+```
+
+After either poetry or pip installation of dependencies modules can be executed as follows:
+
+```bash
+cd PET2BIDS/pypet2bids
+python dcm2niix4pet.py --help
+```
+
+</details>
+
+**Note:**
+*We recommend using dcm2niix v1.0.20220720 or newer; we rely on metadata included in these later releases. It's best to 
+collect releases from the [rorden lab/dcm2niix/releases](https://github.com/rordenlab/dcm2niix/releases) page. We have
+observed that package managers such as yum or apt or apt-get often install much older versions of dcm2niix e.g. 
+v1.0.2017XXXX, v1.0.2020XXXXX. You may run into invalid-BIDS or errors with this software with older versions.* 
+
+
+### spreadsheet_conversion (custom and pmod)
+
+This folder contains spreadsheets templates and examples of metadata and matlab and python code to convert them to json files. Often, metadata such as Frame durations, InjectedRadioactivity, etc are stored in spreadsheets and we have made those function to create json files automatically for 1 or many subjects at once to go with the nifti imaging data. Note, we also have conversion for pmod files (also spreadsheets) allowing to export to blood.tsv files.
+
+### metadata
+
+A small collection of json files for our metadata information. 
+
+### user metadata 
+
+No matter the way you prefer inputting metadata (passing all arguments, using txt or env file, using spreadsheets), you are always right! DICOM values will be ignored - BUT they are checked and the code tells you if there is inconsistency between your inputs and what DICOM says.
```

Comment:
I would suggest to rephrase this part. Something in the way:

User-defined metadata (passed in arguments, defined in txt or spreadsheets) will take priority over values in the DICOM header, but pet2bids will warn you if your values do not coincide with DICOM ones.

Reply:
I'm partial that section being emphatic and PET2BIDS does scream at you with error/warning messages to belabor the point.

Disregard the fact that when running pet2bids I use the `--silent` option.